File: nicegui_tabulator/core/teleport.js
export default {
  template: `<Teleport :to="to" :key="key"><slot></slot></Teleport>`,
  props: {
    to: String,
  },
  data() {
    return {
      key: 0,
    };
  },
  methods: {
    forceUpdate() {
      // console.log('forceUpdate', this.to);
      this.key++;
    }
  }
};

File: nicegui_tabulator/core/teleport.py
from nicegui.element import Element


class Teleport(Element, component="teleport.js"):
    def __init__(self, to: str) -> None:
        """ """
        super().__init__()
        self._props["to"] = to

    def forceUpdate(self):
        self.run_method("forceUpdate")


teleport = Teleport
